client/src/Components/OneVideoCard.js

```diff
@@ -0,0 +1,63 @@
+import { useState } from "react";
+const OneVideoCard = ({ id, title, url, rating, handleDelete }) => {
+  const [videoRating, setVideoRating] = useState(rating);
+
+  const increase = async () => {
+    setVideoRating(videoRating + 1);
+    fetch(`https://video-recomendations-server.onrender.com/videos/${id}`, {
+      method: "PUT",
+      headers: {
+        "Content-Type": "application/json",
+      },
+      body: JSON.stringify({ rating: videoRating + 1 }),
+    })
+      .then((response) => response.json())
+      .then((data) => console.log(data))
+      .catch((error) => console.log(error));
+  };
+  const decrease = () => {
+    if (videoRating > 0) {
+      setVideoRating(videoRating - 1);
+    }
+    fetch(`http://localhost:4000/videos/${id}`, {
+      method: "PUT",
+      headers: {
+        "Content-Type": "application/json",
+      },
+      body: JSON.stringify({ rating: videoRating - 1 }),
```

Comment:
Interesting - in one of these you cap the value at 0, but in the other you let the value go below zero... Is that intentional?